CMakeLists.txt

```diff
@@ -22,6 +22,16 @@ project(
 #
 include(cmake/CompilerWarnings.cmake)
 include(cmake/StandardSettings.cmake)
+find_package(LLVM REQUIRED CONFIG)
```

Comment:
There are already some llvm package config code in https://github.com/eunomia-bpf/bpftime/blob/a509ebf6f3a1260030e39980e1d4c3120b30665b/vm/llvm-jit/CMakeLists.txt#L40

Is it duplicate?

Reply:
I don't think remove the find_package(LLVM REQUIRED CONFIG) from bpftime/vm/llvm-jit/CMakeLists.txt is a good idea.

Imagining a user who still don't want compile llvm-jit and wish to continue using simple jit, then cmake will have errorbecause cannot find llvm install location.